### ISSUE TITLE
Continue training saved model

### ISSUE THREAD

**Issue by aDotInTheVoid** [HyperGAN/HyperGAN#184]
I have the following file's from previous training:

- checkpoint  
- model.ckpt.data-00000-of-00001  
- model.ckpt.index 
- model.ckpt.meta

How can I continue training the model with these files, so I can pick up on the progress saved in them.
The cli documentation is quite unclear on this.

**Comment by MaffooBristol** [comment]
@aDotInTheVoid 
From what I have found personally, if you just start training again whilst those files exist within the same config (eg, `saves/default/checkpoint`), it will carry on. However it will start the sampling again from 00000.jpg so I'd recommend moving your old output images somewhere else if you want to keep them.

Running `hypergan train` again when I have those files will output the following lines:

```
[discriminator] Class loss is off.  Unsupervised learning mode activated.
[hypergan] |= Loading network from /home/ubuntu/foo/saves/default/model.ckpt
[hypergan] |= Loading checkpoint from /home/ubuntu/foo/saves/default
```

I'd keep an eye out for those to check it's doing what you expect. This is all on the latest alpha of 0.10 by the way, I'm not sure about how it functions in the version available on pip

**Comment by martyn** [comment]
Hi, in the recently released 0.10 the save path "saves/#{config}" will be checked for an existing file.  In case of an architecture change, hg performs optimistic loading(restores what it can).

Resuming sampling is forthcoming.
